hermes/templates/deployment.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "hermes.fullname" . }}
+  labels:
+    {{- include "hermes.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "hermes.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "hermes.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - hermes
+            - run
+            - --pythnet-http-endpoint
+            - {{ .Values.hermes.pythnetHttpEndpoint }}
+            - --pythnet-ws-endpoint
+            - {{ .Values.hermes.pythnetWsEndpoint }}
+            - --api-addr
+            - 0.0.0.0:8080
```

Comment:
I am using a fixed port here instead of allowing users to chose it because it is likely they choose 80 and 80 is not allowed with our default security context (that prevents privilege escalation)

Reply:
Keep in mind that the pod port isn't used directly (the service port is and it routes to the pod port), so this is fine.